app/components/User/UserCard.tsx

```diff
@@ -0,0 +1,143 @@
+"use client";
+import Image from 'next/image';
+import Link from 'next/link';
+import { useEffect, useState } from 'react';
+import { FaGithub } from 'react-icons/fa';
```

Comment:
Instead of using FaGithub from 'react-icons' you can use the icon that is already in the repository public/assets/github_icon.svg